rundown/views.py

```diff
@@ -83,4 +85,12 @@ def patch(self, request, id):
         updated_rundown.description = new_description
         updated_rundown.save()
         rundown_serializers = RundownSerializer(updated_rundown)
-        return Response(rundown_serializers.data, status=200) 
\ No newline at end of file
+        return Response(rundown_serializers.data, status=200) 
+
+class RundownListView(generics.ListAPIView):
+    permission_classes = [IsAuthenticated]
```

Comment:
Is isAuthenticated enough for this? I think only the owner should access (event owner that can access this endpoint). What do you think?

Reply:
Yeah, I agree with Alvaro. It is better if only the creator of the event can access this rundown.

Reply:
Will modify!

Reply:
Have modified! Please check @emiriko @JohannesSetiawan 😃